mods/ts/rules/defaults.yaml

```diff
@@ -357,6 +357,16 @@
 	TimedUpgradeBar@EMPDISABLE:
 		Upgrade: empdisable
 		Color: 255,255,255
+	WithDecoration@EMPDISABLE:
+		Image: emp
+		Sequence: idle
+		ReferencePoint: HCenter, VCenter
+		Palette: effect
+		UpgradeTypes: empdisable
+		ShowToEnemies: true
+		ZOffset: 8192
```

Comment:
Considering that I know why it is bogus (WithDecoration does not check for the actor's current ZOffset), I am working on a fix already.